### PR TITLE
fixes tagline  and testimonials update

### PR DESCRIPTION
**Pull request by SevaDer14**
PT: tagline and testimonial bugs (Admin and Client)

### CODE REVIEW

File: src/components/navigation/Sidebar.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const Sidebar = () => {
           <PersonIcon />
         </SidebarIcon>
         <SidebarIcon
-          dataCy='testimonial-creation'
+          dataCy='testimonial-creation'peak
```
Comment (1CIM) [inline src/components/navigation/Sidebar.jsx:35]: What this do?

Comment (SevaDer14) [reply src/components/navigation/Sidebar.jsx:35]: oops

**Review by 1CIM — APPROVED** [review]
LGTM

**Review by lukeperrin10 — APPROVED** [review]
LGTM

**Pull request merged**